### ISSUE TITLE
Setting stratification to Yes automatically sets shape to StoolEntero

### ISSUE THREAD

**Issue by JoFAM** [mafed/shinyMB#1]
If you set Stratification to "YES", you cannot use any other shape but StoolEntero. When trying to change the shape, it switches back to StoolEntero immediately when you go to the tab Abundance Curves.

Is this by design?

**Comment by mafed** [comment]
Yes indeed, this because among the example datasets we have stratification factors only for _"StoolEntero"_.
It's curious though as I didn't remember it was switching back automatically but rather raising an error, maybe this is the effect of some Shiny improvement on automatic error-handling. At least I don't recall hard-coding this behaviour.
